proto/orc_proto.proto

```diff
@@ -178,18 +208,34 @@ message Type {
 }
 
 message StripeInformation {
+  // the global file offset of the start of the stripe
   optional uint64 offset = 1;
+  // the number of bytes of index
   optional uint64 indexLength = 2;
+  // the number of bytes of data
   optional uint64 dataLength = 3;
+  // the number of bytes in the stripe footer
   optional uint64 footerLength = 4;
+  // the number of rows in this stripe
   optional uint64 numberOfRows = 5;
+  // If this is present, it resets the stripe id with respect to encryption
+  // until the next reset. Otherwise, it is the previous stripe id + 1.
+  // This should only occur in the first stripe and the boundaries of
+  // merged ORC files.
+  optional uint64 encryptStripeId = 6;
+  // For each encryption variant, the new encrypted key to use until we
+  // find a replacement.
+  repeated bytes encryptedKeys = 7;
 }
 
 message UserMetadataItem {
   optional string name = 1;
   optional bytes value = 2;
 }
 
+// The ORC 1 stripe statistics are organized as a sequence of
+// StripeStatistics (1 per a stripe), which each contain the
+// ColumnStatistics for each column.
 message StripeStatistics {
```

Comment:
Should we mark it as [deprecated=true] in ORC v2?

Reply:
We can't really deprecate, since we'll need to leave the definition for ORC v0 and v1 readers.

But yes, we should make it as only used in v0 and v1.